### PR TITLE
Update emissions.md to reflect changes from CM_391 related to emissio…

### PR DESCRIPTION
**Pull request by marideeweber**
…n factor outliers

### CODE REVIEW

File: emissions.md [hunk 1]
```diff
@@ -104,6 +104,14 @@ There are some naming conventions for a few emission species/sectors within GCAM
...
 * Sulfur dioxide (SO<sub>2</sub>) emissions are currently differentiated by four metaregions (SO2\_1,SO2\_2,SO2\_3,SO2\_4). These are kept separate for purposes of feeding the emissions information to the climate model; for emissions estimation, these different categories should be added to obtain global totals.
 
+##### Outlier Emission Factors
+
+Outlier air pollutant emission factors are replaced with the global (GCAM) or national (GCAM-USA) median.
+
+* GCAM: Outlier emission factors are identified as being two standard deviations above or three below the national median for a given pollutant, year, supplysector, subsector, and technology. These outliers are replaced with the global median for the given pollutant, year, supplysector, subsector, and technology. This method is applied to replace outliers for all supplysectors in the [CEDS data](https://github.com/JGCRI/CEDS) except for out_resources. The out_resources supplysector was not included since previous work implemented an alternate strategy to address outliers here, as additional processing needed to be done to exclude model regions that had very low resource production and did not yield reliable emission factors.
```
Comment (mbins) [inline emissions.md]: Does "national median for a given pollutant, year, supplysector, subsector, and technology" mean the global median value (e.g., median across nations for a given pollutant, year, supplysector, subsector, and technology), or the median within a nation for a given pollutant, year, supplysector, subsector, and technology?  I suspect it's the former, but the way this reads suggests the latter.

File: emissions.md [hunk 2]
```diff
@@ -104,6 +104,14 @@ There are some naming conventions for a few emission species/sectors within GCAM
...
+
+* GCAM-USA: Outlier emission factors are identified as being two standard deviations above or three below the national median for a given pollutant, year, supplysector, subsector, and technology. These outliers, and emission factors that are missing or infinite, are replaced with the national median for the given pollutant, year, supplysector, subsector, and technology. This method is applied to replace outliers for air pollutants in the GCAM-USA transportation, electricity generation, buildings, and industrial energy use sectors.
```
Comment (mbins) [inline emissions.md]: Suggest "This method is applied to replace outlier emission factors for air pollutants".

**Review by mbins — COMMENTED** [review]
Hi Maridee, I left one question and one suggested edit.  Can you take a look and let me know when this is ready to re-review?

File: emissions.md (hunk 1 above)
Comment (marideeweber) [reply emissions.md]: Typo! It is the global median (across nations), not the national median.

**Comment by mbins** [comment]
I approve this PR.

**Pull request merged**